haystack/document_stores/opensearch.py

```diff
@@ -161,14 +157,15 @@ def __init__(
                                For more information on configuration of knn indices, see
                                [OpenSearch Documentation](https://opensearch.org/docs/latest/search-plugins/knn/knn-index/#method-definitions).
         :param ivf_train_size: Number of embeddings to use for training the IVF index. Training starts automatically
-                               once the number of indexed embeddings exceeds ivf_train_size. If `None`, the minimum
+                               once the number of indelasticsearchexed embeddings exceeds ivf_train_size. If `None`, the minimum
```

Comment:
Not sure what happened here 😅
```suggestion
                               once the number of indexed embeddings exceeds ivf_train_size. If `None`, the minimum
```

Reply:
Weird!! Fixing it :smile: